.github/workflows/build-auto-qttest-on-pullrequest.yml

```diff
@@ -7,56 +7,37 @@ on: [ pull_request ]
 
 env:
   QT_SELECT: 5
-  QT_VERSION: '5.15.2' # quotes required or YAML parser will interpret as float
+  QT_VERSION: '5.15.3' # quotes required or YAML parser will interpret as float
```

Comment:
Last time I checked 5.15.2 is used everywhere else, any reason why using 5.15.3 in the pullrequest?

Reply:
The AppImage was built using 5.15.3. Just wanted to it to match the version used by Jammy.

Reply:
Ah I see, then lets go for this